webCommander/centralHub/views.py

```diff
@@ -51,3 +61,15 @@ def get_artists(request):
         "<option value='{val}'>{val}, Popularity: {pop}</option>".format(val=row[0], pop=row[2]) for row in response
     )
     return HttpResponse(options)
+
+def get_artist_choices(request):
+    print(request.GET.get('artist', ""))
+    name = request.GET.get("artist", "")
+    response = search_spotify_artist(name)
+    options = ''.join(
+        "<option value='{val}'>{val}, Popularity: {pop}</option>".format(val=row[0], pop=row[2]) for row in response
+    )
+
+    print(response)
+    return render(request, 'spotifyFramework/spotify_artist_choices.html',{'response':response})
```

Comment:
here you need to return just the html snippet (concatenated options list)     

```
return HttpResponse(options)
```

I thought we did have this working before?

Reply:
ah I see, that is the method above:

```
def get_artists(request):
    name = request.GET.get("artist", "")
    response = search_spotify_artist(name)

    options = ''.join(
        "<option value='{val}'>{val}, Popularity: {pop}</option>".format(val=row[0], pop=row[2]) for row in response
    )
    return HttpResponse(options)
```

So I am making some redundant changes now. Let's go through this together later ...